src/main/java/ru/lanwen/wiremock/config/WiremockCustomizerException.java

```diff
@@ -0,0 +1,12 @@
+package ru.lanwen.wiremock.config;
+
+/**
+ * {@code WiremockCustomizerException} is the superclass of those
+ * exceptions that can be thrown during the normal customizing operation.
+ *
+ * @author Vincent Palau
+ * @see WiremockCustomizer
+ */
+public class WiremockCustomizerException extends RuntimeException {
```

Comment:
runtime exceptions should not be declared. The whole purpose of the Exception in the signature of the method is to not handle any kind of exceptions internally in the method

Reply:
That makes sense. Changed!